system/handlers/formcontrols/ObjectPicker.cfc

```diff
@@ -64,4 +66,23 @@ component {
 
 		return renderView( view="formcontrols/objectPicker/index", args=args );
 	}
+
+	private string function _removeInvalidValues( required string objectName, required string values ) {
+		if ( !len( args.defaultValue ?: "" ) ) {
+			return "";
+		}
+
+		var initialValues = listToArray( arguments.values );
+		var validValues   = presideObjectService.selectData(
```

Comment:
I _think_ this can just be:

```cfc
var existingValues = ...selectData(...);

return ColumnData( existingValues.id );
```

???

Reply:
Potentially multiple, ordered values. This keeps them in the original order...

Reply:
ah ok, yes.